microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -0,0 +1,155 @@
+
+You can also emulate vision deficiencies by clicking the Eye icon next to the Eyedropper icon.  For example, to emulate blurred vision, click **Emulate Vision Deficiency** (the eye icon, ![Eye icon](./devtools-111-images/eye-device-mode-icon.png)), and then select **Blurred vision**:
+
+![Emulating blurred vision directly from the Device Mode toolbar](./devtools-111-images/new-emulation-options-blurred-vision.png)
+
+See also:
+* [Emulate mobile devices (Device Emulation)](../../../device-mode/index.md)
+* [Check for contrast issues with dark theme and light theme](../../../accessibility/test-dark-mode.md)
+* [Emulate vision deficiencies](../../../accessibility/emulate-vision-deficiencies.md)
+
+
+<!-- ====================================================================== -->
+## Performance tool unminifies file and function names when profiling production code
+<!-- if change heading: update link in what's new item that links to here -->
+
+<!-- Subtitle: As long as the Performance tool can find sourcemaps from Symbol Server, the Performance tool automatically presents the original file and function names in the flame chart. -->
+
+Microsoft Edge 99 added a new feature to the **Performance** tool called **Unminify**.  This **Unminify** feature applied sourcemaps to a profile that was recorded by using the **Performance** tool, and then downloaded the profile to your machine.  The **Unminify** feature is described in [Use your source maps to display original function names in performance profiles](../../2022/03/devtools.md#use-your-source-maps-to-display-original-function-names-in-performance-profiles) in _What's new in DevTools (Microsoft Edge 99)_.
+
+In Microsoft Edge 111, this feature has been removed, because the **Performance** tool now automatically provides the original file and function names while profiling production code, as long as sourcemaps are hosted on Azure Artifacts Symbol Server.
+
+In the following image, you see minified function names in the flame chart when profiling production code.
+
+![Minified function names in the flame chart](./devtools-111-images/minified-flame-chart.png)
+
+Now in Microsoft Edge 111, the **Performance** tool automatically unminifies function names in the flame chart when profiling production code. 
+
+![Unminified function names in the flame chart](./devtools-111-images/unminified-flame-chart.png)
+
+See also:
+* [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md)
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+* [Record performance](../../../evaluate-performance/reference.md#record-performance) in _Performance features reference_.
+
+
+<!-- ====================================================================== -->
+## Styles tab easier to use in Focus Mode
```

Comment:
I'm not super opinionated here but I want us to be consistent. I call the Styles part of the Elements tool the Styles pane, which is a metaphor we borrow from having panels and those panels are composed of panes (but I realize it's not consistent with using the term "tool" which I also do. We can call it the Styles tab if we want to but we should do it everywhere within this announcement. Another option is the Styles pane then has tabs, like: Computed, Event Listeners, Accessibility, etc.


```suggestion
## The Styles pane is easier to use in Focus Mode
```